.github/workflows/publish-npm-package.yml

```
@@ -28,6 +28,7 @@ jobs:
           NODE_ENV=production yarn build
           yarn test
           yarn lint
+          cd packages/injected && yarn lint:dist && cd - 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)